src/app/body/body.component.html

```diff
@@ -0,0 +1,3 @@
+<main class="main">
```

Comment:
You don't need this component, router outlet move to the app component.

Reply:
fixed